generic3g/CMakeLists.txt

```
@@ -58,6 +57,32 @@ esma_add_library(${this}
   DEPENDENCIES MAPL.regridder_mgr MAPL.geom_mgr MAPL.shared MAPL.profiler MAPL.base MAPL.hconfig_utils YAFYAML::yafyaml PFLOGGER::pflogger GFTL_SHARED::gftl-shared-v2 GFTL::gftl-v2
   TYPE SHARED
   )
+
+##### New function to avoid conflicts with files with the same name
+function(mapl_add_fortran_submodules)
```

Renaming macro now to avoid dealing with it when migrated to ESMA_cmake

```suggestion
function(esma_add_fortran_submodules)
```